### PR TITLE
SPLAT-2955 Release v3.15.0

### PR DESCRIPTION
**Pull request by bender-wk**
JIRA: https://jira.atl.workiva.net/browse/SPLAT-2955

@Workiva/product2

* #1823 go-lib: bump github.com/nats-io/nats-server/v2 from 2.7.2 to 2.7.3 in /lib/go
* #1822 java-lib: bump nexus-staging-maven-plugin from 1.6.11 to 1.6.12 in /lib/java
* #1825 java: Add ephemeral property for HTTP request headers

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
<details>
<summary>(5) Vulnerable direct dependencies were detected</summary>
<li>1 vulns in <code>aiohttp < 3.7.4</code> via <code>lib/python/requirements_dev_asyncio.txt</code></li>
<li>1 vulns in <code>github.com/nats-io/nats-server/v2 < 2.7.2</code> via <code>lib/go/go.mod</code></li>
<li>1 vulns in <code>org.apache.thrift:libthrift < 0.14.0</code> via <code>lib/java/pom.xml</code></li>
<li>1 vulns in <code>org.apache.thrift:libthrift < 0.14.0</code> via <code>test/integration/java/frugal-integration-test/pom.xml</code></li>
<li>1 vulns in <code>org.nanohttpd:nanohttpd</code> via <code>test/integration/java/frugal-integration-test/pom.xml</code> with no fix reported by GitHub</li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=frugal">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by danielharasymiw-wf — APPROVED** [review]
QA +1 pending builds pass
All prs in this release have appropriate sign offs.

**Comment by danielharasymiw-wf** [comment]
@Workiva/release-management-p

File: lib/java/src/main/java/com/workiva/frugal/server/FDefaultNettyHttpProcessor.java [hunk 1]
```diff
@@ -161,18 +174,34 @@ public ByteBuf processFrame(ByteBuf inputBuffer) throws TException, IOException
             );
         }
 
+        Map<Object, Object> ephemeralProperties = new HashMap<>();
```
Comment (markmadej-wk) [inline lib/java/src/main/java/com/workiva/frugal/server/FDefaultNettyHttpProcessor.java:177]: I didn't expect any changes that weren't just version bumps in this PR.  Did you intend to commit these changes too?

Comment (danielharasymiw-wf) [reply lib/java/src/main/java/com/workiva/frugal/server/FDefaultNettyHttpProcessor.java:177]: Frugal/messaging changes are first merged into a "develop" branch. That branch is then merged into master with the version bump changes

Comment (markmadej-wk) [reply lib/java/src/main/java/com/workiva/frugal/server/FDefaultNettyHttpProcessor.java:177]: Ah gotcha, thanks for the explanation

**Review by markmadej-wk — APPROVED** [review]

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**